concepticondata/conceptlists/Greenhill-2015-2525.tsv

```diff
@@ -793,7 +793,7 @@ Greenhill-2015-2525-1965	1965	to jump	1829	JUMP
 Greenhill-2015-2525-1966	1966	shoot, plant	3358	SPROUT (SHOOT)
 Greenhill-2015-2525-1967	1967	firelight		
 Greenhill-2015-2525-1968	1968	vulva	1837	VULVA
-Greenhill-2015-2525-1969	1969	short-piece-of-wood		
+Greenhill-2015-2525-1969	1969	short-piece-of-wood	3222	TWIG
```

Comment:
Could it also be a branch? Then 3223 BRANCH OR TWIG would fit better.

Reply:
I think a branch wouldn't be short? but happy to change if you want

Reply:
True. Let's leave the mapping as is.

Reply:
ok